CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# These owners will be the default owners for everything in the repo.
+# Unless a later match takes precedence, they will be requested for review when someone
+# opens a pull request.
+*       @blootsvoets @nivemaham @fnobilia
```

Comment:
Does this mean each pull request needs three approvals? Or one of the three will be enough? Otherwise perhaps we can enable `CODEOWNERS` pull request reviews on `master`, and require at least one approved review on `dev`?